src/components/components.scss

```diff
@@ -12,4 +12,11 @@
   'Navbar/Navbar',
   'Sidebar/Sidebar',
   'TextArea/TextArea',
-  'Hero/Hero';
+  'Hero/Hero',
+  'Builder/Description/Description',
+  'Builder/Newsletter/Newsletter',
+  'Builder/Social/Social',
+  'Builder/Contact/Contact',
+  'Builder/Header/Header',
+  'ListHeader/Description/Description',
+  'ListHeader/Title/Title';
```

Comment:
I dont think we should list these components in a Builder or ListHeader directory. Lets just keep them as components so we can see if there are any collisions, ie you have 2 description components that do identical things, we should keep things DRY and just reuse components